### PR TITLE
Add offline method for get_app_access_token

### PR DESCRIPTION
**Pull request by kz26**
PEP8 fix

PEP8 under-indentation fix

### CODE REVIEW

File: facebook/__init__.py [hunk 1]
```diff
@@ -282,13 +282,23 @@ def fql(self, query):
...
+        """
+        Get the application's access token as a string.
+        If offline=True, use the concatenated app ID and secret
+        instead of making an API call.
+        <https://developers.facebook.com/docs/facebook-login/
+        access-tokens#apptokens>
+        """
+        if offline:
+            return "%s|%s" % (app_id, app_secret)
+        else:
```
Comment (martey) [inline facebook/__init__.py]: Would it make sense to eliminate this "else", since if `offline` is True, the method will immediately return?

**Comment by martey** [comment]
Closing and reopening to force a Travis CI run.

File: facebook/__init__.py (hunk 1 above)
Comment (kz26) [inline facebook/__init__.py]: I guess that's just my personal style for clarity - feel free to remove it if you prefer.

**Comment by martey** [comment]
The commit from this pull request has been merged (see 4f4856b91c12a13b24e62e5e48f9a487bf0c35b5).

**Pull request closed without merging**